bot/src/lib.rs

```diff
@@ -45,5 +45,22 @@ pub fn cron_main(bot_args: BotArgs) -> anyhow::Result<()> {
         // You can edit retries with CW_ORCH_MAX_TX_QUERY_RETRIES
         bot.fetch_contracts()?;
         bot.autocompound();
+
+        // Drop connection
+        let mut state = std::sync::Arc::try_unwrap(bot.daemon.daemon.state).unwrap();
+        drop(state.grpc_channel);
```

Comment:
We are sure that the grpc channel is not used anywhere while being dropped ?

Reply:
I think so! At this point only one Daemon is active, are there any other cases this grpc connection can be active?

Reply:
it could be querying something in the background. Because a channel can be copied easily, this needs to be made sure !

Reply:
> it could be querying something in the background. Because a channel can be copied easily, this needs to be made sure !

Don't think it can be in the process of querying something in the background. It's single-threaded program right now!

Reply:
Yeah it can't work, because Daemon initializes with 2 strong pointers into daemon state, so we can't get ownership of it